src/components/gauge/gauge.tsx

```diff
@@ -0,0 +1,381 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Listen,
+  Prop,
+  State
+} from "@stencil/core";
+
+import { IComponent } from "../common/interfaces";
+
+@Component({
+  shadow: false,
+  styleUrl: "gauge.scss",
+  tag: "gx-gauge"
+})
+export class Gauge implements IComponent {
+  @Element() element: HTMLElement;
+
+  @Event() gxGaugeDidLoad: EventEmitter;
+
+  /**
+   * Property of type Style.
+   * Define if shadow will display or not. Default is disabled.
+   */
+  @Prop() styleShadow = false;
+
+  /**
+   * This property allows you to select the gauge type. _(Circle or Line)_.
+   * Default is linear type.
+   */
+  @Prop() gaugeType: "line" | "circle" = "line";
```

Comment:
How about if we use just `type`?

Reply:
+1

Reply:
@MarkMed Did you consider this suggestion?

Reply:
Ready! 😃